### ISSUE TITLE
Style lines

### ISSUE THREAD

**Issue by pbrickell** [cmaurer/angularjs-nvd3-directives#321]
Is it possible to style the lines on a line chart? In particular I want some lines to be dashed or dotted. TIA

**Comment by LeeMendelowitz** [comment]
I agree that dotted lines would be useful. Under the hood this library uses [nvd3](http://nvd3.org/). I'm looking into whether that library allows for dotted datapoints on line charts, and if it's easy will make a pull request.

**Comment by pbrickell** [comment]
We did find our way to a partial solution. 

It just so happened that our charts have lines in pairs (expected and actual values). Using that fact we created some css...

#attachmenttimes .nv-groups .nv-series {
    stroke-dasharray: 3, 3;
}
#attachmenttimes .nv-groups .nv-series-1 {
    stroke-dasharray: 3, 3;
}
#attachmenttimes .nv-groups .nv-series-3 {
    stroke-dasharray: 3, 3;
}
 etc.

The numbered .nv-series classes are generated by nvd3. We are relying on these being odd numbered and our paired lines. So this works for us but is not a general solution.

**Comment by LeeMendelowitz** [comment]
Thanks! These also seem to work:

http://stackoverflow.com/questions/18530459/nvd3-js-bigger-points-in-a-line-chart
http://stackoverflow.com/questions/27892806/css-styling-of-points-in-figure

**Comment by pbrickell** [comment]
Those are handy too. Thanks.

In our solution I am a bit concerned that we are relying on an implementation detail i.e. the series numbering convention. We do not get any control over the values so are susceptible to change. It works but I'm not proud of it :-.

Cheers,
Paul B.
